src/main/java/br/com/metricminer2/scm/GitRepository.java

```diff
@@ -100,6 +105,9 @@ public SCMRepository info() {
 			String origin = git.getRepository().getConfig().getString("remote", "origin", "url");
 
 			return new SCMRepository(this, origin, path, headId.getName(), lastCommit.getName());
+		} catch (RepositoryNotFoundException rnfe) {
+			log.error(rnfe.getMessage());
+			return null;
```

Comment:
Agreed. I should have deal with this exception in a higher level, in my study. This was the scenario that I returned null: When mining multiple repositories with `allProjectsIn(path)`, if any directory in that path wasn't a repository, the exception was thrown up to RepositoryMining, them all other repository mining was aborted. Then returning null, only that path was aborted with error logged.
A new specific domain RuntimeException inherited Exception could be thrown, or let user deal with normal RuntimeException...